multires-lyrics-search/README.md

```diff
@@ -1,109 +1,180 @@
 
-# Lyrics Live Search: Understanding the Concept of Chunk
-
-[![Jina](https://github.com/jina-ai/jina/blob/master/.github/badges/jina-badge.svg?raw=true  "We fully commit to open-source")](https://get.jina.ai)
+# Lyrics Live Search: Understanding the Concept of Chunks
 
 [![](demo.gif)](https://www.youtube.com/watch?v=GzufeV8AY_w)
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**
 
-- [Use toy data](#use-toy-data)
-- [[Optional] Download full lyrics dataset](#optional-download-full-lyrics-dataset)
-- [Install](#install)
-- [Run](#run)
-- [View in Browser](#view-in-browser)
+- [🐍 Build the app with Python](#---build-the-app-with-python)
+- [Using the search interface and understanding your results](#using-the-search-interface-and-understanding-your-results)
+- [Overview of the files in this example](#overview-of-the-files-in-this-example)
+- [Flow diagram](#flow-diagram)
+- [Next steps, building your own app](#next-steps--building-your-own-app)
+- [🐳 Deploy the prebuild application using Docker](#---deploy-the-prebuild-application-using-docker)
+- [Community](#community)
 - [License](#license)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
+This example shows you how to build a semantic search app powered by [Jina AI](http://www.jina.ai)'s neural search framework.  You can index and search song lyrics using state of art machine learning language models. This example helps teach you about the concept of chunking and it's importance in search systems. Once you've built the example, you can visual how the system is matching inputs to output via a custom website we have built. 
+
+# 🐍 Build the app with Python
+
+These instructions explain how to build the example yourself and deploy it with Python. If you want to skip the building steps and just run the app, check out the Docker section below.  
+
+## Pre requirements:
+1. You have a working Python 3.7 or 3.8 environment. 
+2. We recommend creating a [new python virtual envoriment](https://docs.python.org/3/tutorial/venv.html) to have a clean install of Jina and prevent dependency clashing.   
+3. You have at least 2GB of free space on your hard drive. 
+
+
+## Step 1. Clone the repo and install Jina
```

Comment:
One thought I had while reading this was that presumably the user will try a couple of examples in a row and we are telling them to clone the repo over and over again.

Maybe we can make it more explicit that they should only do this if this is the first time they are running the examples?

Reply:
Ah fair point, the only issue with that I see; is our repo updates so often that a user could end up out of date. So I'd rather they clone the most recent version. What do you think?

Reply:
We could just tell them to pull the most recent master each time?

Reply:
and tell them to clone the first time and pull the master every other time?

Reply:
Yeah maybe like 'clone the repo and ensure that you have the most recent version of the code (if you cloned the repo before make sure to `git fetch origin`